p/db-busradar-nrw/index.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+// DB Busradar NRW app does not allow selecting specific modes of transport to filter results,
+// so the bitmasks had to be determined by querying some stations and looking at the results..
+const products = [
+	{
+		id: 'nationalExpress',
+		mode: 'train',
+		bitmasks: [1],
+		name: 'InterCityExpress',
+		short: 'ICE',
+		default: true
+	},
+	{
+		id: 'national',
+		mode: 'train',
+		bitmasks: [2],
+		name: 'InterCity & EuroCity',
+		short: 'IC/EC',
+		default: true
+	},
+	// 4: not always used, could not be confirmed..
+	{
+		id: 'regionalExp',
+		mode: 'train',
+		bitmasks: [4],
+		name: 'Regionalexpress',
+		short: 'RE',
+		default: true
+	},
+	{
+		id: 'regional',
+		mode: 'train',
+		bitmasks: [8],
+		name: 'Regionalzug',
+		short: 'RB/RE',
+		default: true
+	},
+	// 16: often appears together with 8 even when there are only S-Bahn trains at the station
```

Comment:
As above.

Reply:
The appearance together with 8 may be because of 8 being used for replacement service, even when it's for S-Bahn, for example Dormagen Chempark has a 16 on it's own while the next station, Köln Worringen, has a 16 with an 8 because there's also replacement service at that station (but no usual RB or RE)

Reply:
Even if you don't *exactly* know what 16 stands for, use the closes identifier & description you can come up with. We can improve on it later.

Also, once you add it, please set `default: true`, unless you're very sure that this product is irrelevant in the results.

Reply:
16 should be ok as is, it is clearly S-Bahn. It's just the 8 which is not very obvious, it seems to be for RB+RE+replacement services